src/functions/functionsTreeItem.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
Adding these global eslint-disable flags adds risk to the code base. 

Each of these should be removed and the code should be fixed to comply.